lib/Varien/Data/Collection/Db.php

```diff
@@ -258,7 +261,16 @@ public function getSelectCountSql()
                         return !preg_match($pattern, $clause);
                     });
                 });
-                if (empty($whereUsingJoin)) {
+                if ($this->_bindParams) {
+                    $bindParams = array_map(function ($token) {
+                        return ltrim($token, ':');
+                    }, array_keys($this->_bindParams));
+                    $bindPattern = '/:('.implode('|', $bindParams).')/';
+                    $joinUsingBind = array_filter($leftJoins, function ($table) use ($bindPattern) {
+                        return preg_match($bindPattern, $table['joinCondition']);
```

Comment:
Thanks. This works for me, but if `$table['joinCondition']` is `null` it throws a Deprecation error: `preg_match(): Passing null to parameter #2 ($subject) of type string is deprecated`

Reply:
@colinmollenhour?